vignettes/vignette time series in data.table.Rmd

```diff
@@ -0,0 +1,795 @@
+---

```

Comment:
small note about the file name: no spaces and the format for the others is like `datatable-<topic>.Rmd`, how about `datatable-time-series.Rmd`?

Reply:
or even `datatable-timeseries.Rmd`